tasks/48_follow-the-spy/follow-the-spy.js

```diff
@@ -0,0 +1,21 @@
+const sortCountries = (a, b) => {
+  if (a[1] === b[0]) {
+    return -1;
+  }
+  if (a[0] === b[1]) {
+    return 1;
+  }
+  return 0;
+};
+
+export function followTheSpy(routes) {
+  return routes.sort(sortCountries).reduce((acc, cur, index) => {
```

Comment:
Returning chaining functions with a bunch of complex logic is a bad practice. Please separate it in different lines